Sources/PackageDescription/Version.swift

```diff
@@ -56,7 +56,11 @@ extension Version: Hashable {
         if let build = buildMetadataIdentifier {
             result = (result &* mul) ^ UInt64(build.hashValue)
         }
-        return Int(bitPattern: UInt(result))
+        #if arch(i386) || arch(arm)
```

Comment:
I think this kind of checks shluld be moved lower into some kind of hashing utility to keep higher level things cleaner.